File: backend/OnboardingDiary.Api/DTOs/Dashboard/RecentEntryDto.cs
namespace OnboardingDiary.Api.DTOs.Dashboard;

public class RecentEntryDto
{
    public string Type { get; set; } = string.Empty;
    public int Id { get; set; }
    public string Title { get; set; } = string.Empty;
    public DateTime Date { get; set; }
    public string? Status { get; set; }
}

File: src/main/java/com/codev/onboardingdiary/web/dto/RecentEntryDto.java
package com.codev.onboardingdiary.web.dto;

import com.codev.onboardingdiary.domain.EntryCategory;
import java.time.LocalDate;

public record RecentEntryDto(EntryCategory category,
                             Long id,
                             LocalDate date,
                             String title,
                             String detail,
                             String link) {
}
